fjord/feedback/models.py

```diff
@@ -408,8 +403,103 @@ def infer_platform(cls, product, browser):
         return u''
 
 
-@register_mapping_type
-class ResponseMappingType(FjordMappingType, Indexable):
+class ResponseDocTypeManager(FjordDocTypeManager):
```

Comment:
I decided to model this similarly to the Django Model/ModelManager. I'm not sure if that was helpful or just added unneeded complexity.